server/document.js

```diff
@@ -100,15 +100,20 @@ export class Head extends Component {
 }
 
 export class Main extends Component {
+  static propTypes = {
+    className: PropTypes.string
+  }
+
   static contextTypes = {
     _documentProps: PropTypes.any
   }
 
   render () {
     const { html, errorHtml } = this.context._documentProps
+    const { className } = this.props
     return (
       <Fragment>
-        <div id='__next' dangerouslySetInnerHTML={{ __html: html }} />
+        <div id='__next' dangerouslySetInnerHTML={{ __html: html }} className={className} />
```

Comment:
Lets add a test case for this. So that it won't break in the future 👍

Reply:
I'll have something in today. Thanks!

Reply:
Awesome! Thanks ❤️

Reply:
Apologies. It has taken a bit longer than expected to get to this.

I opened up the `test` folder and was trying to decipher where everything was located. Is it just me, or does everything seem a bit "disorderly" there? At the very least, I was having some trouble getting my bearings.

Any suggestions to which file this test should reside in would be very helpful. :)